webapp/frontend/src/pages/index.tsx

```diff
@@ -1,15 +1,23 @@
+import Layout from "@/components/layout/Layout";
+
 export default function Home() {
```

Comment:
should we still keep this function named `Home()`? Since we have an actual home page now let's change it to be less confusing

Reply:
It's supposed to be kept. It's a page. In NextJs, files in the pages directory contain functions that are pages and not mere react components. Pages contain one or more components that make up the page.